Docs: Add Installation YouTube Videos to documentation pages. (#88465)
grafana/grafana: docs/sources/setup-grafana/installation/redhat-rhel-fedora/index.md

If you install via RPM or the `.tar.gz` file, then you must manually update Grafana for each new version.

> The following video guides you through the steps, common commands, and how to install Grafana in RHEL or Fedora as described in this document:

Reviewer: ```suggestion
The following video demonstrates how to install Grafana on RHEL or Fedora as outlined in this document:
```
